Challenges may be bound to TLS connections, not HTTP sessions (#246)
ietf-wg-privacypass/base-drafts: draft-ietf-privacypass-auth-scheme.md

message matches the concatenation of the client nonce and the hash of a
valid TokenChallenge. For context-bound tokens, origins store or reconstruct the
contexts of previous TokenChallenge structures in order to validate the token.

> A TokenChallenge MAY be bound to a specific TLS connection from a client, but

Reviewer: ```suggestion
A TokenChallenge MAY be bound to a specific TLS session with a client, but
```
Since it would be reasonable for the challenge to be associated with a ticket, for example.
